packages/docs/src/components/DocsFile.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="mb-3">
+    <va-content>
```

Comment:
The entire page has a `va-content` wrapper
```suggestion
```

Reply:
Perhaps it would be convenient to add something like `va-collapse` here to minimize the code block